amber-java-common/src/main/resources/drivermsg.proto

```diff
@@ -35,5 +38,5 @@ message DriverMsg {
     optional uint32 synNum = 3;
     optional uint32 ackNum = 4;
     optional uint32 listenerNum = 5;
-    extensions 10 to 99;
-}
+    extensions 10 to 199;
```

Comment:
@kubicz10 be aware of changing anything in files `*.proto`. The same changes should be added to https://github.com/project-capo/amber-common